mobile.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./mobile.css">
+    <script src="https://kit.fontawesome.com/136dc91701.js" crossorigin="anonymous"></script>
+    <title>mobile</title>
+</head>
+<body>
+    <header>
+        <div>
+            <nav>
+             <ul class="headerTop">
+                <li class="fb"><a href=""><i class="fa-brands fa-facebook-f"></i></a></li>
+                <li class="twit"><a href=""><i class="fa-brands fa-twitter"></i></a></li>
+                <li class="eng"><a href="">English</a></li>
+                <li class="pg"><a href="">My Page</a></li>
+                <li class="ld"><a href="">Logout</a></li>
+             </ul>
+            </nav>
+        </div>
+        <div class="headerBtm">
+            <div class="logoSite">
+                <a href="./index.html"><img class="logo" src="./icon/gentle.jpeg" alt="Logo" onclick="fun()"></a>
+                <b>Ideal <br> Men</b>
+            </div>
+            <nav>
+            <ul  class="mainNavBar">
+                 <li class="about"><a href="">About</a></li>
+                 <li><a href="">Program</a></li>
+                 <li><a href="">Join</a></li>
+                 <li><a href="">Sponsor</a></li>
+                 <li><a href="">News</a></li>
+                 <li class="compain"><a href="">CC Campain</a></li>
+            </ul>
+
+            </nav>
```

Comment:
- [ ] [Optional] In the navbar for the about page, instead of having a link to the about page, kindly change it to home, also for the mobile version.